spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/config/GcpConfigProperties.java

```diff
@@ -31,17 +31,23 @@
 @ConfigurationProperties("spring.cloud.gcp.config")
 public class GcpConfigProperties implements GcpConfigPropertiesProvider {
 
+	/** Enables Spring Cloud GCP Config */
 	private boolean enabled = true;
 
+	/** Name of the application */
 	private String name;
 
+	/** Profile under which app is running (e.g., "prod", "dev", "test", etc.) */
 	private String profile = "default";
 
 	// Config server API time out in milliseconds, default = 1 minute
+	/** Timeout for Google Runtime Configuration API calls */
 	private int timeout = 60000;
 
+	/** Overrides the GCP project ID specified in the Core module */
 	private String projectId;
 
+	/** Overrides the GCP OAuth2 credentials specified in the Core module */
```

Comment:
Strictly speaking there is no need for a description for this. This denotes a "nested group" and there is no use of that description there at the moment. It  doesn't hurt of course :)

Reply:
Ah right, makes sense. Might leave it there just because it couldn't hurt :)
Should I bother adding something to `org.springframework.cloud.gcp.core.Credentials`' fields?

Reply:
Yes. One-to-one mapping is handled as well. Again, please use your IDE to see what the developer experience is, you'll see by yourself (just create an app Spring Cloud GCP in it, and see what the auto-completion experience in `application.properties` is).

Reply:
I just tried to preview the changes on IJ and the nested properties' descriptions don't show up, unfortunately.
Leaving them in just in case other IDEs do show them. :)